boards/arm/mec15xxevb_assy6853/doc/index.rst

```diff
@@ -93,14 +101,14 @@ board. Advanced users may deviate from this recommendation.
 Jumper setting for MEC15xx EVB Assy 6853 Rev A1p0
 =================================================
 
-Power-related jumpers.
+Power-related jumpers
 ----------------------
 
-Power from +5V power brick connected to barrel connector P11 (5.5mm OD, 2.1mm ID)
-JP88 5-6 only
+Power from +5V power brick connected to barrel connector ``P11`` (5.5mm OD, 2.1mm ID)
+``JP88 5-6`` only.
```

Comment:
Might help to use complete sentences, e.g.

    If ... of jumper ... is ..., then ... is connected to ...

Reply:
reworded